assignments/arrays.js

```diff
@@ -63,34 +63,87 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
 
+const findCarById = carId => {
+    for (i = 0; i < inventory.length; i++) {
+       inventory[i].id === carId ? console.log(`Car ${carId} is a ${inventory[i].car_year} ${inventory[i].car_make} ${inventory[i].car_model}.`) : i;
```

Comment:
Very good, that way we do not base our search on index, but on id so it is future proof if array will mutate in future.